server/bin/pbench-server-put-shim-001.py

```diff
@@ -0,0 +1,185 @@
+#!/usr/bin/env python3
```

Comment:
Why do we have prep-shim and put-shim, which seem to be at least mostly the same?

Reply:
I will be removing the prep-shim completely and only put-shim will be available for processing

Reply:
I appreciate your trying to make the review easier, but I think it has left us confused with respect to which files will actually be merged and what their respective names will be.  (At the very least, it would be helpful if you could update the PR Description with some of these details.)

Reply:
So, all the comments that we made on prep-shim will need to be applied to put-shim.  :-)

Reply:
Sorry for the confusion. I will update the description